packages/edit-post/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -43,10 +57,49 @@ export default function ListViewSidebar() {
 			onKeyDown={ closeOnEscape }
 		>
 			<div
-				className="edit-post-editor__list-view-panel-header"
+				className="edit-post-editor__list-view-panel-header components-panel__header edit-post-sidebar__panel-tabs"
 				ref={ headerFocusReturnRef }
 			>
-				<strong id={ labelId }>{ __( 'List View' ) }</strong>
+				<ul>
```

Comment:
This should appear after the Close button, not before for discovery reasons.

Reply:
Applied the feedback 👍

Reply:
This does make the tab order not match the visual order, and there's also now an inconsistency with the settings sidebar and how its tabs and close button are ordered.

Reply:
> This does make the tab order not match the visual order

Yep, I just noticed this and it's a valid concern. I'll create a new issue to report other a11y improvement as well.